src/main/scala/cromwell/engine/EngineFunctions.scala

```diff
@@ -1,25 +1,16 @@
 package cromwell.engine
 
 import cromwell.binding.WdlFunctions
-import cromwell.binding.values.{WdlString, WdlValue}
+import cromwell.binding.values.{WdlInteger, WdlString, WdlValue}
+import cromwell.engine.backend.local.TaskExecutionContext
 
-class EngineFunctions(val ctx: TaskExecutionContext) extends WdlFunctions {
-  def getFunction(name: String): WdlFunction = {
-
-    def read_int(params: Seq[WdlValue]): WdlValue = {
-      // TODO: Validate parameters
+import scala.util.Try
 
-      // Just assume the first parameter is a String (example only)
-      val path = params.head.asInstanceOf[WdlString].value
-
-      // For 3-step workflow, path == "stdout"
-
-      // Use ctx along with path to construct WdlValue
-      ???
-    }
-
-    def read_string(params: Seq[WdlValue]): WdlValue = ???
+abstract class EngineFunctions(val ctx: TaskExecutionContext) extends WdlFunctions {
```

Comment:
Just to get my comment in writing... TaskExecutionContext should probably be defined in the backend and this abstract class (now trait?) shouldn't TaskExecutionContext